Dockerfile.base

```diff
@@ -21,7 +21,8 @@ RUN apt-get update && \
     apt-transport-https \
     ca-certificates \
     sudo \
-    gnupg-agent \
+    gpg-agent \
```

Comment:
unfortunately this breaks ubuntu xenial and others

```
E: Unable to locate package gpg-agent
```

It might be worth creating a dockerfile specifically for debian buster to make these changes not get out of control per base

id be ok with a mostly duplicated `Dockerfile.base.debian` thats committed instead of generated off the `Dockerfile.base` now that more differences are showing